Sources/SwiftDocC/Utility/MarkupExtensions/ListItemExtractor.swift

```diff
@@ -49,7 +49,7 @@ let simpleListItemTags = [
 ]
 
 extension Sequence where Element == InlineMarkup {
-    private func splitNameAndContent() -> (name: String, nameRange: SourceRange?, content: [Markup], range: SourceRange?)? {
+    private func splitNameAndContent(skipStartingTag: Bool = false) -> (name: String, nameRange: SourceRange?, content: [Markup], range: SourceRange?)? {
```

Comment:
Can you explain why is this parameter needed? If the code scans up to the first colon then does it matter if the line looks like
```
- one two: Description 
```
or 
```
- one: Description 
```
?